RawCMS.Library/Lambdas/JsDispatcher.cs

```diff
@@ -28,23 +29,115 @@ public JsDispatcher(EntityService entityService)
             this.entityService = entityService;
         }
 
-        public override void Execute(string collection, ref JObject item, ref Dictionary<string, object> dataContext)
+        public void ExecuteInternal(string collection, ref JObject item, ref Dictionary<string, object> dataContext, PipelineStage stage, DataOperation Operation)
         {
+            var eventName = Stage.ToString() + Operation.ToString();
```

Comment:
There is a mismatch from UI managed entity structure for events and execution from BE. For example on UI the entity structure now is something like this:
```
{
    "CollectionName": "Prova",
    "FieldSettings": [
        {
            "Options": {},
            "Name": "Prova",
            "Type": "text"
        }
    ],
    "PresaveScript": "console.log('presave');",
    "PostsaveScript": "console.log('postsave');",
    "PreDeleteScript": "console.log('predelete');",
    "PostDeleteScript": "console.log('postdelete');"
}
```
But the correct structure is 

```
{
    "CollectionName": "Prova",
    "FieldSettings": [
        {
            "Options": {},
            "Name": "Prova",
            "Type": "text"
        }
    ],
   "Events": {
        "PreOperationWrite": "console.log('presave');",
        "PostOperationWrite": "console.log('postsave');",
        "PreOperationDelete": "console.log('predelete');",
        "PostOperationDelete": "console.log('postdelete');"
    }
}
```